docs/src/pages/resources.tsx

```diff
@@ -52,76 +54,43 @@ const installationLinks = [
   ],
 ];
 
-const additionalResources = [
-  [
-    'YouTube Channel',
-    'https://www.youtube.com/channel/UCMuwrvBsg_jjI2gLcm04R0g',
-  ],
-  [
-    'May 15, 2020: Virtual Meetup Recording. Topics: 0.36 Overview, Committers Self-Intro, Roadmap',
-    'https://www.youtube.com/watch?v=f6up5x_iRbI',
-  ],
-  [
-    "O'Reilly Apache Superset Quick Start Guide",
-    'https://www.oreilly.com/library/view/apache-superset-quick/9781788992244/',
-  ],
-  [
-    'Getting Started with Apache Superset, an Enterprise-Ready Business Intelligence Platform',
-    'https://reflectivedata.com/getting-started-apache-superset-enterprise-ready-business-intelligence-platform/',
-  ],
-  [
-    'Unlocking Advanced Data Analytics on The Data Lake Using Apache Superset and Dremio',
-    'https://www.dremio.com/tutorials/dremio-apache-superset/',
-    {
-      sub: 'Dremio',
-      link: 'https://www.dremio.com',
-    },
-  ],
-  [
-    'Test-driving Apache Superset',
-    'https://blog.smartcat.io/2018/test-driving-apache-superset/',
-    {
-      sub: 'SmartCat',
-      link: 'https://smartcat.io',
-    },
-  ],
-  [
-    'Build Apache Superset from source',
-    'https://hackernoon.com/a-better-guide-to-build-apache-superset-from-source-6f2ki32n0',
-  ],
-];
-
 const youtubeRefs = [
-  "https://www.youtube.com/embed/24XDOkGJrEY",
-  "https://www.youtube.com/embed/AqousXQ7YHw",
-  "https://www.youtube.com/embed/JGeIHrQYhIs",
-  "https://www.youtube.com/embed/z350Gbi463I"
+  'https://www.youtube.com/embed/24XDOkGJrEY',
+  'https://www.youtube.com/embed/AqousXQ7YHw',
+  'https://www.youtube.com/embed/JGeIHrQYhIs',
+  'https://www.youtube.com/embed/z350Gbi463I',
 ];
 
 const youtubeIds = [
   [
     0,
```

Comment:
These are the indices we can skip, as per the comment further down.

Reply:
I don't think this array is necessary. The youtubeIds data structure below could include or reconstruct these strings.